File: students/students/src/MON.SPA/src/models/studentClass/studentClassSummaryModel.js
export class StudentClassSummaryModel{
    constructor(obj = {}){
        this.schoolYear = obj?.schoolYear || 0;
        this.institutionId = obj?.institutionId || 0;
        this.institutionName = obj?.institutionName || 'НЯМА';
        this.className = obj?.className || 'НЯМА';
        this.basicClassName = obj?.basicClassName || '';
        this.classId = obj?.classId || null;
        this.classTypeName = obj?.classTypeName || '';
        this.classSpecialityName = obj?.classSpecialityName || '';
        this.classEduFormName = obj?.classEduFormName || '';
        this.isCurrent = obj.isCurrent || false;
        this.status = obj.status || 0;
        this.enrollmentDate = (obj?.enrollmentDate && this.$moment) ? this.$moment(obj.enrollmentDate).format("MM/DD/YYYY") : '';
    }
}

File: students/students/src/MON.Models/StudentModels/StudentClassSummaryModel.cs
namespace MON.Models.StudentModels
{
    using System;

    public class StudentClassSummaryModel
    {
        public int ClassId { get; set; }
        public int SchoolYear { get; set; }
        public int InstitutionId { get; set; }
        public string InstitutionName { get; set; }
        public int StudentClassId { get; set; }
        public string ClassName { get; set; }
        public int BasicClassId { get; set; }
        public string BasicClassName { get; set; }
        public string ClassTypeName { get; set; }
        public string ClassSpecialityName { get; set; }
        public int ClassEduFormId { get; set; }
        public string ClassEduFormName { get; set; }
        public DateTime EnrollmentDate { get; set; }
        public bool? IsCurrent { get; set; }
        public int? Status { get; set; }
        public int? PositionId { get; set; }
        public string Position { get; set; }
        public int? ClassKind { get; set; }
    }
}
